RELEASE-NOTES.md

```diff
@@ -1,8 +1,24 @@
 # Wikibase DataModel release notes
 
-## Version 7.0.0 (dev)
-
-* Removed `FingerprintHolder`
+## Version 7.0.0 (2017-03-07)
+
+This release allows `EntityIdValue` to support custom entity types, and needs to change snak,
+qualifier, and reference hashes because of this.
```

Comment:
This release adds support for custom entity types to `EntityIdValue`, and thus to changes the hashes of snaks, qualifiers, and references.